de_web_plugin.cpp

```diff
@@ -1181,6 +1181,16 @@ void DeRestPluginPrivate::addLightNode(const deCONZ::Node *node)
                     }
                     break;
 
+                case DEV_ID_XIAOMI_SMART_PLUG:
```

Comment:
I think in this case it's safer to extend the filter with `model id` to pick the Xiaomi plug.


According the 0xFFFF device identifier defined by DEV_ID_XIAOMI_SMART_PLUG this is tricky since device identifiers from *0xC000&ndash;0xFFFF* fall into the reserved range and normally shouldn't be used in HA or ZLL profile. I figure there are more devices than the Xiomi smart plug with that device id.

Hence DEV_ID_RESERVED_FFFF or some other general name might be a better for that define.

Reply:
I wasn't sure `modelId` is already available this early in `addLightNode()`.

I figured the test on `deviceId`, _VENDOR_JENNIC_, endpoint 0x01 and `node->nodeDescriptor().receiverOnWhenIdle()` would probably be safe for now.

Reply:
>The combined test is on deviceId, VENDOR_JENNIC, endpoint 0x01, OnOff cluster, and node->nodeDescriptor().receiverOnWhenIdle(). I figured that would probably be safe for now.

Agree, a small comment should be sufficient in this case.